initializers/virtual_machines.yml

```diff
@@ -0,0 +1,9 @@
+#- cluster: cluster1
```

Comment:
could you please provide one example containing the following fields: `role`, `cluster group`, `tenant group`